packages/atom-light-ui/index.less

```diff
@@ -0,0 +1,22 @@
+
+// Atom Dark UI theme
```

Comment:
[<img width="250" alt="screen shot 2018-09-25 at 10 13 10 am" src="https://user-images.githubusercontent.com/2988/46020150-b1385600-c0ab-11e8-82e0-02eeef443d46.png">](https://twitter.com/jasonrudolph/status/988451756890841088)

:joy:

Reply:
Haha, nice, I'll fix that ;)